pytket/tests/wasm-generation/wasmfromcpp/README.md

```diff
@@ -0,0 +1,22 @@
+Ways to compile wasm from C/C++ are:
+
+```
+emcc <filename>.c -o <filename>.html
+```
+
+```
+clang --target=wasm32 --no-standard-libraries -Wl,--export-all -Wl,--no-entry -o <filename>.wasm <filename>.c
+```
+
+```
+clang --target=wasm32 -mmultivalue -Xclang -target-abi -Xclang experimental-mv --no-standard-libraries -Wl,--export-all -Wl,--no-entry -o <filename>.wasm <filename>.c
```

Comment:
Any reason not to always run this command, i.e. do we need to also give the version without multivalue support?

Reply:
This command is not working with older clang versions, I thought it might be nice to have both. If you thing we should remove this, I am happy to do that.

Reply:
I have added some details to the readme in [b07edfa](https://github.com/CQCL/tket/pull/1070/commits/b07edfaf94711c39f792a056a2f98daa4a3e057c)